crates/coin-store/migrations/001_initial.sql

```diff
@@ -18,7 +18,31 @@ CREATE TABLE blinder_keys (
     FOREIGN KEY (txid, vout) REFERENCES utxos(txid, vout)
 );
 
-CREATE INDEX idx_utxos_asset_id ON utxos(asset_id);
-CREATE INDEX idx_utxos_is_spent ON utxos(is_spent);
-CREATE INDEX idx_utxos_script_pubkey ON utxos(script_pubkey);
-CREATE INDEX idx_utxos_asset_spent_value ON utxos(asset_id, is_spent, value DESC);
+CREATE TABLE assets
+(
+    asset_name TEXT NOT NULL,
+    asset_id   BLOB NOT NULL,
+    PRIMARY KEY (asset_name)
+);
+
+CREATE TABLE option_params
+(
+    script_pubkey           BLOB NOT NULL PRIMARY KEY,
+    start_time              BLOB NOT NULL,
+    expiry_time             BLOB NOT NULL,
+    collateral_per_contract BLOB NOT NULL,
+    settlement_per_contract BLOB NOT NULL,
+    collateral_asset_id     BLOB NOT NULL,
+    settlement_asset_id     BLOB NOT NULL,
+    option_token_entropy    BLOB NOT NULL,
+    grantor_token_entropy   BLOB NOT NULL
+);
```

Comment:
This is bad design.....

Reply:
Let's discuss it offline, though in a nutshell this module should not be responsible for that